modules/displayMessage.js

```diff
@@ -0,0 +1,28 @@
+import commentsCounter from './commentsCounter.js';
+// eslint-disable-next-line no-undef
+const commentsPoke = document.querySelector('.comments');
+const value = document.querySelector('.messageCount');
+const displayMessage = (array2) => {
+  let str = '';
+  // eslint-disable-next-line no-restricted-syntax, no-undef
+  try {
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
Please don't disable the linters. Consider fixing the linter issues instead of disabling the linters.